demisto_sdk/commands/format/update_generic.py

```diff
@@ -370,3 +373,9 @@ def initiate_file_validator(self) -> int:
 
             else:
                 return SUCCESS_RETURN_CODE
+
+    def sync_data_to_master(self):
+        data_from_master = get_remote_file(self.relative_content_path, github_repo=GitContentConfig.OFFICIAL_CONTENT_REPO_NAME)
```

Comment:
In the contribution flow `prev_ver='xsoar-contrib/master'`.
@ShahafBenYakir wdyt? should `content/master` be ok, or should it be configurable?

Reply:
I think it's ok as:
a. The command could potentially run on other repos.
b. The PR opened is between the new branch and **content master** so it make more sense for it be identical to it.